android/app/src/main/res/values/strings.xml

```diff
@@ -6,5 +6,4 @@
   <string name="exposure_notification_channel_name">Exposure Notification</string>
   <string name="exposure_notification_message">Someone you were near has tested positive for COVID-19. Tap for more info.</string>
   <string name="exposure_notification_title">Possible COVID-19 exposure</string>
-  <string name="background_worker_notification_title">Checking exposures…</string>
```

Comment:
@mattThousand @juancruzgs this string keeps getting removed during ./src/locales/pull.sh runs, but is referenced in the code: https://github.com/Path-Check/gaen-mobile/blob/bcc8b881df4a43651694d2ffb2e1d4ff053ef8ba/android/app/src/main/java/org/pathcheck/covidsafepaths/exposurenotifications/common/NotificationHelper.java#L66

Reply:
good catch @artgibson, will fix